### ISSUE TITLE
Bento Clone: Accordion content section should have accessibility roles

### ISSUE THREAD

**Issue by krdwan** [ampproject/amphtml#31137]
From a11y audit, an issue with `amp-accordion` has been tracked.  The issue was resolved for `amp` (0.1) component.  This issue is created to track for `bento` (1.0) component.

The `amp` issue (0.1) with full details can be found here: https://github.com/ampproject/amphtml/issues/30727
